build/make.go

```diff
@@ -105,6 +90,16 @@ func compileGauge() {
 	runProcess("go", args...)
 }
 
+func signDarwinExecutable(execPath string) {
+	id := os.Getenv("OS_SIGNING_IDENTITY")
+	if id == "" {
+		log.Printf("No singning identity found . Executable won't be signed.")
+	}
+	if getGOOS() == "darwin" {
```

Comment:
This can be the first check. No need to check for `OS_SIGNING_IDENTITY` if it is not `darwin` right?